start-site/src/test/java/io/spring/start/site/extension/dependency/springcloud/SpringCloudStreamBuildCustomizerTests.java

```diff
@@ -85,6 +89,19 @@ void springCloudStreamWithKafkaStreams(Version springBootVersion, Dependency tes
 			.hasDependenciesSize(5);
 	}
 
+	@ParameterizedTest
+	@MethodSource("springCloudStreamWithPulsarArguments")
```

Comment:
I am not super excited of screaming "...WithPulsar" here but the variations we are now trying to test are:
- Boot 2.7.x (the test binder deps differ across 2.x and 3.x)
- Boot 3.x (test test binder deps differ across 2.x and 3.x)
- Boot 3.2.0-M3++ (new binder was added here)

Because the Pulsar binder does not exist <=3.2.0-M3 we can simply add more args to the test params for boot32x as the behavior is additive (iow new binder dep).

Reply:
I am not a big fan of those either. I've polished to make it short for now. We can revisit those tests once we remove support for 2.x which is soon.